Library/Formula/supervisor.rb

```diff
@@ -0,0 +1,133 @@
+  end
+
+  resource "elementree" do
+    url "http://effbot.org/media/downloads/elementtree-1.2.6-20050316.tar.gz"
+    sha256 "b29d5f2417cb331562c8c5f8ebef2a895ba540261c4245f526143c6a31bccb04"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+
+    resources.each do |r|
+      r.stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    # Can't use `setup_install_args` - http://github.com/Supervisor/supervisor/issues/608
+    system "python", "setup.py", "install", "--prefix=#{libexec}", "--record=installed.txt"
+
+    bin.install Dir[libexec/"bin/supervisor*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    (prefix/"supervisord.conf").write supervisord_conf
+  end
+
+  def post_install
+    (etc/"supervisord/conf.d").mkpath unless File.directory? etc/"supervisord/conf.d"
+    cp prefix+"supervisord.conf", etc unless File.exist? etc/"supervisord.conf"
+  end
+
+  def caveats; <<-EOS.undent
+    The supervisorctl command will need to be run with the full path to the conf file:
+        supervisorctl -c /usr/local/etc/supervisord.conf
+    You can alias the command in your shell to make it easier:
+        alias supervisorctl='supervisorctl -c #{etc}/supervisord.conf'
+    or with root access link the config to /etc:
+        ln -s #{etc}/supervisord.conf /etc/supervisord.conf
+
+    Add your supervisor program config to: (http://supervisord.org/subprocess.html)
+        #{etc}/supervisord/conf.d/
+    EOS
+  end
+
+  test do
+    system("#{bin}/supervisord --version") # =~ /3\.1\.3/
+    system("#{bin}/supervisorctl --help") # =~ /help/
+  end
+
+  def supervisord_conf; <<-EOS.undent
+    [unix_http_server]
+    file=#{var}/run/supervisor.sock
+    chmod       = 0700
+    ;username    = admin
+    ;password    = somepass
+
+    ;[inet_http_server]
+    ;port        = 127.0.0.1:9001
+    ;username    = admin
+    ;password    = somepass
+
+    [supervisord]
+    logfile     = #{var}/log/supervisord.log
+    logfile_maxbytes = 50MB
+    logfile_backups = 10
+    loglevel    = info
+    pidfile     = #{var}/run/supervisord.pid
+    nodaemon    = false
+    minfds      = 1024
+    minprocs    = 200
+    umask       = 022
+    identifier   = supervisor
+    ;directory   = #{var}/run
+    ;nocleanup   = true
+    ;childlogdir = /usr/local/var/log
+
+    [supervisorctl]
+    serverurl   = unix://#{var}/run/supervisor.sock
+    ;serverurl   = http://127.0.0.1:9001
+    ;username    = admin
+    ;password    = somepass
+
+    [rpcinterface:supervisor]
+    supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+    [include]
+    files = #{etc}/supervisor.d/*.conf
+    EOS
+  end
+
+  # vars don't seem to be available here
+  plist_options :manual => "supervisord -c /usr/local/etc/supervisord.conf"
```

Comment:
` -c #{HOMEBREW_PREFIX}/etc/supervisord.conf `, or ` {etc}/supervisord.conf `. Homebrew isn't always installed in ` /usr/local ` despite urging.